tst/working/veryslow/MatFDPM.tst

```diff
@@ -1,33 +1,42 @@
 # FDPM = fully deleted permutation module
-
-# Alternating:
-gap> deg := Random(10,88);;
-gap> g := AlternatingGroup(deg);;
-gap> f := GF(Random([2,3,4,5,7,9,11]));;
-gap> gens := List([1..5],x->PseudoRandom(g));;
-gap> mgens := List(gens,x->PermutationMat(x,deg,f));;
-gap> m := GModuleByMats(mgens,f);;
-gap> cf := MTX.CompositionFactors(m);;
-gap> dims := List(cf,x->x.dimension);;
-gap> max := Maximum(dims);;
-gap> pos := Position(dims,max);;
-gap> x := PseudoRandom(GL(max,f));;
-gap> mgens := List(cf[pos].generators,y->y^x);;
-gap> h := Group(mgens);;
-gap> ri := RECOG.TestGroup(h,false,Size(g));;
-
 # Symmetric:
-gap> deg := Random(10,88);;
-gap> g := SymmetricGroup(deg);;
-gap> f := GF(Random([2,3,4,5,7,9,11]));;
-gap> gens := List([1..5],x->PseudoRandom(g));;
-gap> mgens := List(gens,x->PermutationMat(x,deg,f));;
-gap> m := GModuleByMats(mgens,f);;
-gap> cf := MTX.CompositionFactors(m);;
-gap> dims := List(cf,x->x.dimension);;
-gap> max := Maximum(dims);;
-gap> pos := Position(dims,max);;
-gap> x := PseudoRandom(GL(max,f));;
-gap> mgens := List(cf[pos].generators,y->y^x);;
-gap> h := Group(mgens);;
-gap> ri := RECOG.TestGroup(h,false,Size(g));;
+gap> testFDPM := function(deg, field, alternating)
+> local g, gens, mgens, m, cf, dims, max, pos, x, h, result;
+> if alternating then
+>   g := AlternatingGroup(deg);
+> else
+>   g := SymmetricGroup(deg);
+> fi;
+> gens := List([1..5],x->PseudoRandom(g));
+> mgens := List(gens,x->PermutationMat(x,deg,field));
+> m := GModuleByMats(mgens,field);
+> cf := MTX.CompositionFactors(m);
+> dims := List(cf,x->x.dimension);
+> max := Maximum(dims);
+> pos := Position(dims,max);
+> x := PseudoRandom(GL(max,field));
+> mgens := List(cf[pos].generators,y->y^x);
+> h := Group(mgens);
+> return Size(RecogniseGroup(h)) = Size(g);
+> #return RECOG.TestGroup(h,false,Size(g));
+> end;;
+gap> l := [];
+gap> for deg in [10, 13, 17, 32] do
+> for field in List([2,3,4,7,9], GF) do
```

Comment:
Why have you removed the fields of size 5 and 11?

Reply:
Because I think the old tests were a bit excessive. I just removed a bit here and there. We'll have to double check with the codecov bot whether this leads to us covering less code.

Reply:
I'll have to resolve the conflicts first.